Resources/views/Block/timeline.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 
 {% block block %}
     <div class="box box-primary">
-        {% if settings.title %}
+        {% if settings.title and settings.title is not empty %}
```

Comment:
After reflexion, I think `settings.title` and `settings.title is not empty` are partially the same, second is more complete.

Why not just keep the second condition?